### PR TITLE
[Feature](inverted index) add lowercase option for inverted index analyzer

### PR DESCRIPTION
**Pull request by airborne12**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by airborne12** [comment]
run buildall

**Comment by airborne12** [comment]
run buildall

File: docs/zh-CN/docs/data-table/index/inverted-index.md [hunk 1]
```diff
@@ -90,6 +90,9 @@ Doris倒排索引的功能简要介绍如下：
     - ignore_above：控制字符串是否建索引。
       - 长度超过 ignore_above 设置的字符串不会被索引。对于字符串数组，ignore_above 将分别应用于每个数组元素，长度超过 ignore_above 的字符串元素将不被索引。
       - 默认为 256 字节
+    - lowercase: 是否将分词进行小写转换，从而在匹配的时候实现忽略大小写
```
Comment (xiaokang) [inline docs/zh-CN/docs/data-table/index/inverted-index.md:93]: keep consistent with code 'lower_case'

File: be/src/olap/rowset/segment_v2/inverted_index_writer.cpp [hunk 2]
```diff
@@ -192,6 +192,12 @@ class InvertedIndexColumnWriterImpl : public InvertedIndexColumnWriter {
             // ANALYSER_NOT_SET, ANALYSER_NONE use default SimpleAnalyzer
             _analyzer = std::make_unique<lucene::analysis::SimpleAnalyzer<char>>();
         }
+        auto lowercase = get_parser_lowercase_from_properties(_index_meta->properties());
+        if (lowercase == "true") {
+            _analyzer->set_lowercase(true);
+        } else if (lowercase == "false") {
+            _analyzer->set_lowercase(false);
+        }
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:200]: what about else lowercase == "" if use have not set this property?

Comment (airborne12) [reply be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:200]: if not set, use different default value for different tokenizer

Comment (xiaokang) [reply be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:200]: get, you can add a comment here

**Comment by airborne12** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Review by zzzxl1993 — APPROVED** [review]
LGTM

**Pull request merged**